### PR TITLE
Pre-processes the Query, to remove trailing semicolon

### PR DESCRIPTION
**Pull request by Sonalisinghal**
Fixes Issue #49

### CODE REVIEW

File: moz_sql_parser/__init__.py [hunk 1]
```diff
@@ -32,6 +32,9 @@ def __deploy__():
 
 def parse(sql):
     try:
+        sql = sql.rstrip()
+        if sql[-1]==";":
+            sql=sql[:-1]
```
Comment (klahnakoski) [inline moz_sql_parser/__init__.py]: instead, write
 
    sql = sql.rstrip().rstrip(";") 

or maybe 

    sql = sql.rstrip("\n\t ;")

https://docs.python.org/2.7/library/string.html#string.rstrip

Comment (Sonalisinghal) [reply moz_sql_parser/__init__.py]: Updated @klahnakoski

File: moz_sql_parser/__init__.py [hunk 2]
```diff
@@ -32,6 +32,7 @@ def __deploy__():
 
 def parse(sql):
     try:
+        sql = sql.rstrip().restrip(";")
```
Comment (klahnakoski) [inline moz_sql_parser/__init__.py]: the tests failed (example https://travis-ci.org/mozilla/moz-sql-parser/jobs/499841236) .  Notice the spelling mistake: `restrip`

Comment (Sonalisinghal) [reply moz_sql_parser/__init__.py]: Sorry for the typo, wouldn't happen again :)

Updated

**Pull request merged**